### PR TITLE
argoproj.github.io/workflows

### PR DESCRIPTION
**Pull request by aslafy-z**
package for https://argoproj.github.io/workflows/ cli tool

### CODE REVIEW

File: projects/argoproj.github.io/workflows/package.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - install dist/argo-linux-amd64 {{prefix}}/bin/argo
+  env:
+    GIT_TAG: "v{{version}}"
+    VERSION: "v{{version}}"
+    RELEASE_TAG: "true"
+    GIT_COMMIT:
+    GIT_TREE_STATE:
+    GOARGS:
+    GOOS:
+    GOARCH:
```
Comment (aslafy-z) [inline projects/argoproj.github.io/workflows/package.yml:23]:
```suggestion
    GOARGS: FOO=bar
```

File: projects/argoproj.github.io/workflows/package.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+distributable:
+  url: https://github.com/argoproj/argo-workflows/archive/v{{version}}.tar.gz
+  strip-components: 1
+display-name: argo-workflows
+versions:
+  github: argoproj/argo-workflows
+build:
+  dependencies:
+    go.dev: '*'
+    git-scm.org: '*'
```
Comment (aslafy-z) [inline projects/argoproj.github.io/workflows/package.yml:10]:
```suggestion
```

File: projects/argoproj.github.io/workflows/package.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+build:
+  dependencies:
+    go.dev: '*'
+  script:
+    - make GIT_TAG="v{{version}}" RELEASE_TAG=true STATIC_FILES=false GIT_COMMIT= GIT_TREE_STATE= GOARGS= dist/argo-linux-amd64
+    - mkdir -p {{prefix}}/bin
+    - install dist/argo-linux-amd64 {{prefix}}/bin/argo
+  env:
+    LDFLAGS:
```
Comment (aslafy-z) [inline projects/argoproj.github.io/workflows/package.yml:18]:
```suggestion
    LDFLAGS: ""
```

**Comment by aslafy-z** [comment]
@jhheider thank you! Any plans to add the add the hacktoberfest label to the repository?

**Comment by jhheider** [comment]
Good question. Let me consult the team.

**Pull request merged**